addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/RFXComBindingConstants.java

```diff
@@ -72,8 +73,7 @@
     public static final String CHANNEL_RAIN_TOTAL = "rainTotal";
     public static final String CHANNEL_WIND_DIRECTION = "windDirection";
     public static final String CHANNEL_WIND_SPEED = "windSpeed";
-    public static final String CHANNEL_GUST = "gust";
-    public static final String CHANNEL_CHILL_FACTOR = "chillFactor";
```

Comment:
Where these unused and not defined somewhere else, where they also should be removed?